vignettes/creating-EML.Rmd

```diff
@@ -191,56 +234,93 @@ The `add_maintenance` function allows you to append the status of the dataset to
 your file and the inputs of `complete` and `ongoing` are the only ones allowed.
 If the dataset is still in progress, the frequency of which it is updated must 
 be provided as well. This information should be added to the "maintenance" tab 
-in the excel file "template.xlsx".
+in the excel file "example-metadata.xlsx".
 
 ```{r}
-maintenance <- read_excel("data-raw/template/template.xlsx", sheet = "maintenance")
-parent_element <- add_maintenance(parent_element = parent_element, status = maintenance$status,
+maintenance <- metadata_sheets$maintenance
+parent_element <- add_maintenance(parent_element = parent_element, 
+                                  status = maintenance$status,
                                   update_frequency = maintenance$update_frequency)
 ```
 
-### Dataset Methods
-Overall methods can be appended next. The `add_method` function allows you to
-append the different scientific methods used in the collection process of your data.
-Methods are appended in a similar fashion as the abstract. A template methods document
-is given ("hannon_example_methods.docx"). If more methods are needed, you can create 
-seperate word documents and provide all of the file paths in the "template.xlsx" 
-excel sheet. The methods can be applied here as well as in the data table element,
-so make sure you are appending the correct methods to the accurate places. 
-Top level methods can describe how a particular csv data was collected, while a data table
-method can explain how particular attributes in the data table were collected.
+### Project: Title, Personnel, and Funding 
+The project section should be appended next. Project personnel and project funding 
+are nested in this project section. To generate a valid EML doc the project section 
+must contain a project title, a project personnel, and project funding. 
 
+#### Project title
 ```{r}
-method_table <- read_excel("data-raw/template/template.xlsx", sheet = "methods")
-for (i in 1:nrow(method_table)) {
-  current <- method_table[i, ]
-  new_method <- add_method(parent_element = parent_element, methods_file = current$methods_file,
-                           instrumentation = current$instrumentation)
-  if (is.null(parent_element$methods)) {
-    parent_element$methods <- new_method
-  } else {
-    parent_element$methods <- list(parent_element$methods, new_method)
-  }
+project_title <- metadata_sheets$project$project_title
+```
+
+#### Project personnel 
+```{r}
+project_personnel_table <- metadata_sheets$project
+for (i in 1:nrow(project_personnel_table)) {
+  current <- project_personnel_table[i, ]
+  project_personnel <- add_personnel(parent_element = list(), 
```

Comment:
I get what's happening, namely you are passing in an empty "parent" to make this function "create" a personal entity but not add it, you add it to the actual data below. I think this is fine for now, but it might worth thinking about creating either a new function based on `add_personnel` maybe called `add_project_personnel` that handles the creation of the person and adds it to the correct spot on the "parent", this is what `add_personnel` currently does

Reply:
Makes sense. I will think about the best approach.